lambdas/service/app.py

```diff
@@ -925,7 +929,80 @@ def generate_manifest(event, context):
     return {'Location': presigned_url}
 
 
-@app.route('/dss/files/{uuid}', methods=['GET'], cors=True)
+dss_files_spec = {
+    'description': 'Request a signed download link to a file from the Data Store',
```

Comment:
We should make the distinction of the `/fetch/dss/files/{uuid}` and `/dss/files/{uuid} ` endpoint descriptions since it is a significant one.